tests/test_hep_model.py

```diff
@@ -34,3 +37,28 @@ def test_special_collections_map_contains_all_valid_special_collections():
     result = SPECIAL_COLLECTIONS_MAP.keys()
 
     assert sorted(expected) == sorted(result)
+
+
+def test_curated():
+    schema = load_schema('hep')
+    subschema = schema['properties']['curated']
+
+    # Can't create a record from an empty snippet, so added this dummy content
+    snippet = (
+        '<datafield tag="020" ind1=" " ind2=" ">'
+        '  <subfield code="a">9780198759713</subfield>'
+        '</datafield>'
+    )  # record/1510325
+    expected_curated = True
+    result = hep.do(create_record(snippet))
+
+    assert 'curated' in result
+    assert validate(result['curated'], subschema) is None
+    assert expected_curated == result['curated']
+
+    expected = [
+        {'a': '9780198759713'},
+    ]
```

Comment:
I should probably check the whole record, to make sure it did not add a key from the curated flag right?

Reply:
@jacquerie ^ maybe you have a better idea...